pig/src/main/java/org/jboss/pnc/bacon/pig/impl/repo/ParentPomDownloader.java

```diff
@@ -56,12 +57,13 @@ private void process() throws IOException {
     }
 
     private void doProcess() throws IOException {
-        Files.walk(repoPath)
-                .filter(Files::isRegularFile)
-                .filter(ParentPomDownloader::isPom)
-                .map(Pom::new)
-                .filter(Pom::hasParent)
-                .forEach(p -> processPom(repoPath, p));
+        try (Stream<Path> stream = Files.walk(repoPath)) {
+            stream.filter(Files::isRegularFile)
```

Comment:
@michalszynkiewicz @thescouser89 See, it aligned this one, but why doesn't it align it with the first dot? And why didn't it align the other statement?

Reply:
Because it does it only if it exceeds the max line length, not always.